README.md

```diff
@@ -0,0 +1,50 @@
+## Ticket booking application
+
+Ticket booking application is a prototype of an application to make reservations on screenings in cinema.
+
+### How to start
+In order to run Ticket booking application two gradle tasks need to be started: *gradlew build* and then *gradlew bootRun*.
+
+### Demos
+1. Shell script to build and run the application (file *build-run.sh*) is in *src/main/resources* directory.
+2. During an application start it's initially populated with test data set.
+This data set is read from *src/main/resources/data.csv* file. 
+3. To see basic use case scenario, run shell script *src/main/resources/use-case-scenario.sh*. 
+The script is using **jq** library in order to make work with JSON's easier.
+Library is available under [jq github page](https://stedolan.github.io/jq).
+
+### Continuous integration
+In order to provide continuous integration I have added integration with Travis CI. 
+
+### Additional assumptions
+
+#### Screenings
+* The system lists movies available after given time by user. 
+Listing contains screenings starting from the rounded given hour 
+and ending with the end of the day. 
+Example: When user gives time '2020-01-04 15:30' (formatted with ISO-8601 standard)
```

Comment:
to nie jest ISO-8601 - brakuje `T`

Reply:
Prawda. Nie zapisałam daty z 'T' bo wydawało mi się to bardziej przejrzyste. Po przemyśleniu faktycznie błędne założenie i w README powinna być data zapisana w formacie ISO-8601.